src/styles/platform-navigator.scss

```diff
@@ -0,0 +1,58 @@
+.platform-navigator {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  &__button {
+    display: flex;
+    justify-content: space-between;
+  }
+  .popover {
+    left:0;
+    z-index:1;
+  }
+
+  &__dropdown__item--current {
+    background-color: var(--amplify-colors-teal-20);
+  }
+}
+
+.info-popover {
+  .popover {
+    width: 22rem;
+    left: auto;
+    right: auto;
+    top: calc(100% + 1rem);
+    margin-left: -10.5rem;
```

Comment:
Sorry Jacob, we might need to update this again? I'm wondering if instead of `margin-left` we could also just use `left` to specify how much we want to move?

I removed `margin-left` and set `left: 25%`, how does this look to you?

(I think you have to remove the `left` on the popover in your tsx file too for it to work)